cdap-ui/app/cdap/components/PluginJSONCreator/Create/Content/JsonMenu/index.tsx

```diff
@@ -38,46 +45,108 @@ const JsonMenuView: React.FC<ICreateContext> = ({
   jsonView,
   setJsonView,
   outputName,
+  setPluginState,
 }) => {
+  const pluginData = {
+    pluginName,
+    pluginType,
+    displayName,
+    emitAlerts,
+    emitErrors,
+    configurationGroups,
+    groupToInfo,
+    groupToWidgets,
+    widgetToInfo,
+    widgetToAttributes,
+    outputName,
+  };
+
+  const [JSONStatus, setJSONStatus] = React.useState(JSONStatusMessage.Pending);
+  const [JSONErrorMessage, setJSONErrorMessage] = React.useState('');
+
+  // When JSON error occurs, show the error message for 2 seconds.
```

Comment:
This will give no time for the user to read the message, understand and process. The pattern we already have is to show the error message and allow the user to close the message. Success messages can be transient as they provide only confirmations and not information.

Reply:
<img width="1792" alt="Screen Shot 2020-06-05 at 9 39 19 AM" src="https://user-images.githubusercontent.com/14116152/83882663-70159780-a710-11ea-9598-c703a1248ce1.png">

I've set error banner that appears. My previous approach was for error message to show for only few seconds. This new UI pattern will ensure user to have enough time to read the error message. 

Resolved.